docs/dump_javadump.md

```diff
@@ -925,7 +927,242 @@ Help > Install New Software > Work with "IBM Diagnostic Tool Framework for Java"
 ```
 
 If, unlike the previous scenario, you receive an `OutOfMemoryError` and the MEMINFO section shows that there is very little space left
-on the Java heap, the current thread information is not important. The current thread is simply the thread that happened to be current when the space ran out. In this situation you might want to increase your Java heap size. For help with this task, see [How to do heap sizing](https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.vm.80.doc/docs/mm_heapsizing.html).
+on the Java heap, the current thread information is typically not important. The current thread is simply the thread that happened to be current when the space ran out. In this situation you might want to increase your Java heap size. For help with this task, see [How to do heap sizing](https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.vm.80.doc/docs/mm_heapsizing.html).
```

Comment:
> (https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.vm.80.doc/docs/mm_heapsizing.html).

Is there an OpenJ9 link to this content yet?

Reply:
Not yet but it is something we need to prioritise because it crops up so regularly. I've started some work on the GC section. I'll try and get something in there for heap sizing and put this up for general  review. (There is *so* much content we need to write for GC, we'll just have to start somewhere and grow it over time.)